openjdk/mmtkHeap.cpp

```diff
@@ -77,6 +77,12 @@ jint MMTkHeap::initialize() {
   size_t mmtk_heap_size = heap_size;
   /*forcefully*/ //mmtk_heap_size = (1<<31) -1;
 
+  // Set options
+  if (ThirdPartyHeapOptions != NULL) {
```

Comment:
Once we set the default value to be an empty string, we can get rid of this null check.